src/ts/inMemoriam.ts

```diff
@@ -183,15 +183,13 @@ export class InMemoriam {
         select.value = (typeof filters[select.name] !== 'undefined' ? filters[select.name] : '');
       }
 
-      Events.addEventHandler(select, 'change', this.eventHandlers[select.id]);
+      Events.addEventHandler(select, 'change', this._eventHandlers[select.id]);
     });
 
   }
 
   private clearMarkerCluster(): this {
-    if (this.markerCluster) {
-      this.markerCluster.clearMarkers();
-    }
+    if (this._markerCluster) this._markerCluster.clearMarkers();
```

Comment:
I don't like the single statement syntax since it's ugly and is complexing the code reading unnecessarily :/